01-101.Rmd

```diff
@@ -437,7 +437,13 @@ echo "Hello, again!"
 ottrpal::include_slide("https://docs.google.com/presentation/d/1BQxrVYdKZTbpCaF-i_q9w7s9x034lEXpQZDU-Sl09cs/edit#slide=id.gff2211b72f_1_58")
 ```
 
-Let's get a bit more advanced. We can load a preconfigured software bundle called a module. This is very convenient because it means we don't need to install anything manually! In this example, we will load a module containing [R](https://www.r-project.org/) version 4.2.0.
+
+## Using Pre-Installed Software Modules
+Let's get a bit more advanced. We can load a preconfigured software bundle called a module. At the Fred Hutch new software modules you commonly use can be requested at any time by emailing SciComp and providing information about the software itself.  
```

Comment:
Not sure this belongs in a beginner course - let's save it for later!

Reply:
maybe we can link out to "more info"?

Reply:
Good point.  A load of people roll their own here b/c they didn't realize they could ask for more software beyond what is listed!!! Is there a way we can say something about being able to ask for them here and then add this more detailed blurb to the FAQ?